afs/afs-storage-api/src/test/java/com/powsybl/afs/storage/AbstractAppStorageTest.java

```diff
@@ -173,9 +172,9 @@ public void test() throws IOException, InterruptedException {
         NodeInfo testDataInfo = storage.createNode(testFolderInfo.getId(), "data", DATA_FILE_CLASS, "", 0, new NodeGenericMetadata());
         NodeInfo testData2Info = storage.createNode(testFolderInfo.getId(), "data2", DATA_FILE_CLASS, "", 0,
                 new NodeGenericMetadata().setString("s1", "v1")
-                                         .setDouble("d1", 1d)
-                                         .setInt("i1", 2)
-                                         .setBoolean("b1", false));
+                        .setDouble("d1", 1d)
```

Comment:
Could you restore the original formatting as it is not a change related to this PR?

Reply:
sry, autoformating file